### ISSUE TITLE
help wanted: find route with dpids and put the route inside l2_multi

### ISSUE THREAD

**Issue by Maryam-Maya-Karimi** [noxrepo/pox#153]
Dear MurphyMc

I find out that I may use it in the method "_get_path" like:
"path=core.Routing.calculate_route(src, dst)" (which is my method for routing and return a string of dpids as route)
instead of
"path = _get_raw_path(src, dst)" (that is an original l2_multi path)
when I print the results here they both show me some thing like:
[00-00-00-00-00-03, 00-00-00-00-00-02, 00-00-00-00-00-01]

but there is a problem.
in "class WaitingPath " in init method when it needs to calculate "self.first_switch = path[0][0].dpid" it gives me an error " 'str' object has no attribute 'dpid'" even when I change the path elements to dpid by str_to_dpid(...) I got the error "" 'int' object has no attribute 'dpid'""
I printed my result in init I saw that my path elements will look like
[('00-00-00-00-00-03', 1, None), ('00-00-00-00-00-06', None, None), (00-00-00-00-00-05, None, 2)]
while the original l2_multi will look like
init [(00-00-00-00-00-03, 1, 2), (00-00-00-00-00-02, 3, 2), (00-00-00-00-00-01, 2, 1)]
while the only change I made was putting my route in _get_path as I mentioned and it seems correct there.
my algorithm calculate route and out put pure string of dpids. can you please help me to put it inside the l2_multi

my question is a continue on this issue: https://github.com/noxrepo/pox/issues/152#issuecomment-120614719 which is closed

thank you very much

**Comment by MurphyMc** [comment]
You could have reused the same issue.

I think _get_raw_path() should return paths of actual Connection objects referring to switches, not just their DPIDs.

See "Getting a Reference to a Connection Object" in the manual for information on getting a Connection object using its DPID.

**Comment by Maryam-Maya-Karimi** [comment]
Dear MurphyMc
I have 2 questions:
1-in l2_multi it is written that "you should make your topology more static." what doea it mean? how is the topology dynamic? does this mean that with every link changes the whole topology will be built from the first?

2- I have a topology with 2 hosts, 7 switches, 7 links. the nodes s1,s2 and s3 are all connected to each other at first and the hosts connected to  s1 and s3 ping each other successfully through path{s1,s3} at first. I omitted the link between s1 and s3. there is another path between s1 and s3 which is {s1,s2,s3}. but it can not find it. do you know the reason?
( I also tried 2 other things if it helps:
1-when I attach the link (s1-s3) after disconnecting it, they started to ping each other again.
2-if I build my topology with 7switches, 6 links with out the link between s1-s3 from the first they can ping each other via {s1,s2,s3} )
I also used the module for spanning tree in pox (import pox.openflow.spanning_tree
    pox.openflow.spanning_tree.launch(True , True)), can it be the reason? if yes why? I need to know the reason of unsuccessful pings.

thank you in advance

**Comment by MurphyMc** [comment]
1) What I meant there is that in a lot of cases, I don't think topology discovery is actually appropriate, and it's actually better to read in the topology from a configuration file and then, optionally, verify it that the physical topology seems to match.  A middle ground is detecting the topology _once_, but then making it an explicit operator action to actually _change_ it, and differentiating between links which are down and links which are absent (to the current code, those are the same thing).

2) What does this topology actually look like?  Run your experiment with POX at DEBUG log level and post the results.

**Comment by Maryam-Maya-Karimi** [comment]
my topology in mininet is:
h8 h8-eth0: 
h9 h9-eth0: 
s1 lo:  s1-eth1:  s1-eth2:s2-eth2 s1-eth3:s3-eth3 s1-eth4:s4-eth2 s1-eth5:s7-eth2
s2 lo:  s2-eth1:  s2-eth2:s1-eth2 s2-eth3:s3-eth2
s3 lo:  s3-eth1:  s3-eth2:s2-eth3 s3-eth3:s1-eth3 s3-eth4:s6-eth3
s4 lo:  s4-eth1:  s4-eth2:s1-eth4
s5 lo:  s5-eth1:  s5-eth2:s6-eth2
s6 lo:  s6-eth1:  s6-eth2:s5-eth2 s6-eth3:s3-eth4
s7 lo:  s7-eth1:  s7-eth2:s1-eth5
c0

I first omit the link between s5 and s6 and then reconnect it. it doesnt' have any problem 
then I omit the link between s1-s3 and it fails to ping each other

here is the log form pox-dart:

maryam@maryam-VPCEA36FG:~/pox-dart-nochanges$ ./pox.py startnx log
POX 0.3.0 (dart) / Copyright 2011-2014 James McCauley, et al.
INFO:host_tracker:host_tracker ready
INFO:misc.full_payload:Requesting full packet payloads
WARNING:core:Still waiting on 1 component(s)
INFO:core:POX 0.3.0 (dart) is up.
INFO:openflow.of_01:[00-00-00-00-00-02 6] closed
INFO:openflow.of_01:[00-00-00-00-00-04 3] connected
INFO:openflow.of_01:[00-00-00-00-00-05 2] connected
INFO:openflow.of_01:[00-00-00-00-00-03 4] connected
INFO:openflow.of_01:[00-00-00-00-00-07 5] connected
INFO:openflow.of_01:[00-00-00-00-00-01 8] connected
INFO:openflow.of_01:[00-00-00-00-00-06 7] connected
INFO:openflow.of_01:[00-00-00-00-00-03 4] closed
INFO:openflow.of_01:[00-00-00-00-00-02 9] connected
INFO:openflow.of_01:[00-00-00-00-00-04 3] closed
INFO:openflow.of_01:[00-00-00-00-00-03 10] connected
INFO:openflow.of_01:[00-00-00-00-00-05 2] closed
INFO:openflow.of_01:[00-00-00-00-00-04 11] connected
INFO:openflow.of_01:[00-00-00-00-00-06 7] closed
INFO:openflow.of_01:[00-00-00-00-00-05 12] connected
INFO:openflow.of_01:[00-00-00-00-00-07 5] closed
INFO:openflow.of_01:[00-00-00-00-00-06 13] connected
INFO:openflow.of_01:[00-00-00-00-00-07 14] connected
INFO:host_tracker:Learned 1 1 d2:47:07:b5:3e:7c
WARNING:forwarding.l2_multi:Not flooding -- holddown active
WARNING:forwarding.l2_multi:Not flooding -- holddown active
INFO:host_tracker:Learned 3 1 9a:d2:8d:d6:63:59
WARNING:forwarding.l2_multi:Not flooding -- holddown active
WARNING:forwarding.l2_multi:Not flooding -- holddown active
WARNING:forwarding.l2_multi:Not flooding -- holddown active
INFO:host_tracker:Learned 1 1 d2:47:07:b5:3e:7c got IP 10.10.10.1
WARNING:forwarding.l2_multi:Not flooding -- holddown active
INFO:openflow.discovery:link detected: 00-00-00-00-00-01.4 -> 00-00-00-00-00-04.2
INFO:openflow.discovery:link detected: 00-00-00-00-00-01.2 -> 00-00-00-00-00-02.2
WARNING:forwarding.l2_multi:Not flooding -- holddown active
WARNING:forwarding.l2_multi:Not flooding -- holddown active
WARNING:forwarding.l2_multi:Not flooding -- holddown active
INFO:openflow.discovery:link detected: 00-00-00-00-00-01.5 -> 00-00-00-00-00-07.2
WARNING:forwarding.l2_multi:Not flooding -- holddown active
WARNING:forwarding.l2_multi:Not flooding -- holddown active
WARNING:forwarding.l2_multi:Not flooding -- holddown active
INFO:openflow.discovery:link detected: 00-00-00-00-00-01.3 -> 00-00-00-00-00-03.3
WARNING:forwarding.l2_multi:Not flooding -- holddown active
INFO:openflow.discovery:link detected: 00-00-00-00-00-02.2 -> 00-00-00-00-00-01.2
WARNING:forwarding.l2_multi:Not flooding -- holddown active
WARNING:forwarding.l2_multi:Not flooding -- holddown active
WARNING:forwarding.l2_multi:Not flooding -- holddown active
INFO:openflow.discovery:link detected: 00-00-00-00-00-02.3 -> 00-00-00-00-00-03.2
WARNING:forwarding.l2_multi:Not flooding -- holddown active
INFO:openflow.discovery:link detected: 00-00-00-00-00-03.4 -> 00-00-00-00-00-06.3
INFO:openflow.discovery:link detected: 00-00-00-00-00-03.2 -> 00-00-00-00-00-02.3
WARNING:forwarding.l2_multi:Not flooding -- holddown active
INFO:openflow.discovery:link detected: 00-00-00-00-00-03.3 -> 00-00-00-00-00-01.3
INFO:openflow.discovery:link detected: 00-00-00-00-00-04.2 -> 00-00-00-00-00-01.4
WARNING:forwarding.l2_multi:Not flooding -- holddown active
INFO:openflow.discovery:link detected: 00-00-00-00-00-05.2 -> 00-00-00-00-00-06.2
INFO:openflow.discovery:link detected: 00-00-00-00-00-06.2 -> 00-00-00-00-00-05.2
INFO:openflow.discovery:link detected: 00-00-00-00-00-06.3 -> 00-00-00-00-00-03.4
INFO:openflow.spanning_tree:6 ports changed
INFO:openflow.spanning_tree:2 ports changed
INFO:openflow.spanning_tree:3 ports changed
INFO:openflow.spanning_tree:3 ports changed
INFO:openflow.discovery:link detected: 00-00-00-00-00-07.2 -> 00-00-00-00-00-01.5
INFO:openflow.spanning_tree:3 ports changed
INFO:openflow.spanning_tree:2 ports changed
INFO:host_tracker:Learned 3 1 9a:d2:8d:d6:63:59 got IP 10.10.10.2
INFO:openflow.discovery:link timeout: 00-00-00-00-00-05.2 -> 00-00-00-00-00-06.2
INFO:openflow.discovery:link detected: 00-00-00-00-00-05.2 -> 00-00-00-00-00-06.2
INFO:openflow.discovery:link timeout: 00-00-00-00-00-03.3 -> 00-00-00-00-00-01.3
WARNING:forwarding.l2_multi:Packet from 9a:d2:8d:d6:63:59 to known destination d2:47:07:b5:3e:7c arrived at 00-00-00-00-00-01.3 without flow
INFO:openflow.discovery:link detected: 00-00-00-00-00-03.3 -> 00-00-00-00-00-01.3

and here is the out put in mininet terminal:
root@maryam-VPCEA36FG:~/OpenNet/mininet/examples# ./avali.py
**\* Adding controller
**\* Starting network
**\* Configuring hosts
h8 h9 
**\* Starting controller
c0 
**\* Starting 7 switches
s1 s2 s3 s4 s5 s6 s7 
Testing network connectivity
**\* h8 : ('ping 10.10.10.2 -c 5',)
PING 10.10.10.2 (10.10.10.2) 56(84) bytes of data.
From 10.10.10.1 icmp_seq=1 Destination Host Unreachable
From 10.10.10.1 icmp_seq=2 Destination Host Unreachable
From 10.10.10.1 icmp_seq=3 Destination Host Unreachable
From 10.10.10.1 icmp_seq=4 Destination Host Unreachable
From 10.10.10.1 icmp_seq=5 Destination Host Unreachable

--- 10.10.10.2 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 4000ms
pipe 4
**\* h8 : ('ping 10.10.10.2 -c 20',)
PING 10.10.10.2 (10.10.10.2) 56(84) bytes of data.
64 bytes from 10.10.10.2: icmp_seq=1 ttl=64 time=176 ms
64 bytes from 10.10.10.2: icmp_seq=2 ttl=64 time=8.21 ms
64 bytes from 10.10.10.2: icmp_seq=3 ttl=64 time=8.22 ms
64 bytes from 10.10.10.2: icmp_seq=4 ttl=64 time=8.54 ms
64 bytes from 10.10.10.2: icmp_seq=5 ttl=64 time=12.3 ms
64 bytes from 10.10.10.2: icmp_seq=6 ttl=64 time=7.92 ms
64 bytes from 10.10.10.2: icmp_seq=7 ttl=64 time=8.22 ms
64 bytes from 10.10.10.2: icmp_seq=8 ttl=64 time=7.92 ms
64 bytes from 10.10.10.2: icmp_seq=9 ttl=64 time=6.83 ms
64 bytes from 10.10.10.2: icmp_seq=10 ttl=64 time=7.86 ms
64 bytes from 10.10.10.2: icmp_seq=11 ttl=64 time=7.28 ms
64 bytes from 10.10.10.2: icmp_seq=12 ttl=64 time=5.75 ms
64 bytes from 10.10.10.2: icmp_seq=13 ttl=64 time=8.48 ms
64 bytes from 10.10.10.2: icmp_seq=14 ttl=64 time=11.3 ms
64 bytes from 10.10.10.2: icmp_seq=15 ttl=64 time=7.58 ms
64 bytes from 10.10.10.2: icmp_seq=16 ttl=64 time=7.24 ms
64 bytes from 10.10.10.2: icmp_seq=17 ttl=64 time=8.15 ms
64 bytes from 10.10.10.2: icmp_seq=18 ttl=64 time=8.51 ms
64 bytes from 10.10.10.2: icmp_seq=19 ttl=64 time=7.61 ms
64 bytes from 10.10.10.2: icmp_seq=20 ttl=64 time=7.84 ms

--- 10.10.10.2 ping statistics ---
20 packets transmitted, 20 received, 0% packet loss, time 19029ms
rtt min/avg/max/mdev = 5.757/16.613/176.290/36.658 ms
port s5-eth2 down
**\* s5 : ('ifconfig s5-eth2 down',)
**\* h8 : ('ping 10.10.10.2 -c 20',)
PING 10.10.10.2 (10.10.10.2) 56(84) bytes of data.
64 bytes from 10.10.10.2: icmp_seq=1 ttl=64 time=8.37 ms
64 bytes from 10.10.10.2: icmp_seq=2 ttl=64 time=7.43 ms
64 bytes from 10.10.10.2: icmp_seq=3 ttl=64 time=9.04 ms
64 bytes from 10.10.10.2: icmp_seq=4 ttl=64 time=7.52 ms
64 bytes from 10.10.10.2: icmp_seq=5 ttl=64 time=10.7 ms
64 bytes from 10.10.10.2: icmp_seq=6 ttl=64 time=7.21 ms
64 bytes from 10.10.10.2: icmp_seq=7 ttl=64 time=7.20 ms
64 bytes from 10.10.10.2: icmp_seq=8 ttl=64 time=7.78 ms
64 bytes from 10.10.10.2: icmp_seq=9 ttl=64 time=142 ms
64 bytes from 10.10.10.2: icmp_seq=10 ttl=64 time=6.71 ms
64 bytes from 10.10.10.2: icmp_seq=11 ttl=64 time=7.76 ms
64 bytes from 10.10.10.2: icmp_seq=12 ttl=64 time=6.99 ms
64 bytes from 10.10.10.2: icmp_seq=13 ttl=64 time=8.46 ms
64 bytes from 10.10.10.2: icmp_seq=14 ttl=64 time=9.45 ms
64 bytes from 10.10.10.2: icmp_seq=15 ttl=64 time=7.99 ms
64 bytes from 10.10.10.2: icmp_seq=16 ttl=64 time=7.75 ms
64 bytes from 10.10.10.2: icmp_seq=17 ttl=64 time=7.25 ms
64 bytes from 10.10.10.2: icmp_seq=18 ttl=64 time=8.81 ms
64 bytes from 10.10.10.2: icmp_seq=19 ttl=64 time=9.49 ms
64 bytes from 10.10.10.2: icmp_seq=20 ttl=64 time=9.38 ms

--- 10.10.10.2 ping statistics ---
20 packets transmitted, 20 received, 0% packet loss, time 19030ms
rtt min/avg/max/mdev = 6.714/14.885/142.274/29.242 ms
port s5-eth2 up
**\* s5 : ('ifconfig s5-eth2 up',)
**\* h8 : ('ping 10.10.10.2 -c 20',)
PING 10.10.10.2 (10.10.10.2) 56(84) bytes of data.
64 bytes from 10.10.10.2: icmp_seq=1 ttl=64 time=8.80 ms
64 bytes from 10.10.10.2: icmp_seq=2 ttl=64 time=8.83 ms
64 bytes from 10.10.10.2: icmp_seq=3 ttl=64 time=7.01 ms
64 bytes from 10.10.10.2: icmp_seq=4 ttl=64 time=7.48 ms
64 bytes from 10.10.10.2: icmp_seq=5 ttl=64 time=8.25 ms
64 bytes from 10.10.10.2: icmp_seq=6 ttl=64 time=167 ms
64 bytes from 10.10.10.2: icmp_seq=7 ttl=64 time=8.80 ms
64 bytes from 10.10.10.2: icmp_seq=8 ttl=64 time=11.6 ms
64 bytes from 10.10.10.2: icmp_seq=9 ttl=64 time=7.78 ms
64 bytes from 10.10.10.2: icmp_seq=10 ttl=64 time=7.35 ms
64 bytes from 10.10.10.2: icmp_seq=11 ttl=64 time=7.91 ms
64 bytes from 10.10.10.2: icmp_seq=12 ttl=64 time=8.56 ms
64 bytes from 10.10.10.2: icmp_seq=13 ttl=64 time=8.40 ms
64 bytes from 10.10.10.2: icmp_seq=14 ttl=64 time=11.9 ms
64 bytes from 10.10.10.2: icmp_seq=15 ttl=64 time=8.21 ms
64 bytes from 10.10.10.2: icmp_seq=16 ttl=64 time=8.86 ms
64 bytes from 10.10.10.2: icmp_seq=17 ttl=64 time=8.66 ms
64 bytes from 10.10.10.2: icmp_seq=18 ttl=64 time=9.85 ms
64 bytes from 10.10.10.2: icmp_seq=19 ttl=64 time=8.40 ms
64 bytes from 10.10.10.2: icmp_seq=20 ttl=64 time=6.50 ms

--- 10.10.10.2 ping statistics ---
20 packets transmitted, 20 received, 0% packet loss, time 19030ms
rtt min/avg/max/mdev = 6.501/16.522/167.245/34.602 ms
port s3-eth3 down
**\* s3 : ('ifconfig s3-eth3 down',)
**\* h8 : ('ping 10.10.10.2 -c 5',)
PING 10.10.10.2 (10.10.10.2) 56(84) bytes of data.

--- 10.10.10.2 ping statistics ---
5 packets transmitted, 0 received, 100% packet loss, time 3999ms

**\* h8 : ('ping 10.10.10.2 -c 20',)
PING 10.10.10.2 (10.10.10.2) 56(84) bytes of data.
From 10.10.10.1 icmp_seq=11 Destination Host Unreachable
From 10.10.10.1 icmp_seq=12 Destination Host Unreachable
From 10.10.10.1 icmp_seq=13 Destination Host Unreachable
From 10.10.10.1 icmp_seq=14 Destination Host Unreachable
From 10.10.10.1 icmp_seq=15 Destination Host Unreachable
From 10.10.10.1 icmp_seq=16 Destination Host Unreachable
From 10.10.10.1 icmp_seq=17 Destination Host Unreachable
From 10.10.10.1 icmp_seq=18 Destination Host Unreachable
From 10.10.10.1 icmp_seq=19 Destination Host Unreachable
From 10.10.10.1 icmp_seq=20 Destination Host Unreachable

--- 10.10.10.2 ping statistics ---
20 packets transmitted, 0 received, +10 errors, 100% packet loss, time 19118ms
pipe 3
port s3-eth3 up
**\* s3 : ('ifconfig s3-eth3 up',)
**\* h8 : ('ping 10.10.10.2 -c 20',)
PING 10.10.10.2 (10.10.10.2) 56(84) bytes of data.
64 bytes from 10.10.10.2: icmp_seq=1 ttl=64 time=225 ms
64 bytes from 10.10.10.2: icmp_seq=2 ttl=64 time=7.99 ms
64 bytes from 10.10.10.2: icmp_seq=3 ttl=64 time=6.84 ms
64 bytes from 10.10.10.2: icmp_seq=4 ttl=64 time=166 ms
64 bytes from 10.10.10.2: icmp_seq=5 ttl=64 time=5.57 ms
64 bytes from 10.10.10.2: icmp_seq=6 ttl=64 time=10.5 ms
64 bytes from 10.10.10.2: icmp_seq=7 ttl=64 time=5.42 ms
64 bytes from 10.10.10.2: icmp_seq=8 ttl=64 time=7.57 ms
64 bytes from 10.10.10.2: icmp_seq=9 ttl=64 time=8.03 ms
64 bytes from 10.10.10.2: icmp_seq=10 ttl=64 time=13.7 ms
64 bytes from 10.10.10.2: icmp_seq=11 ttl=64 time=7.56 ms
64 bytes from 10.10.10.2: icmp_seq=12 ttl=64 time=5.15 ms
64 bytes from 10.10.10.2: icmp_seq=13 ttl=64 time=7.19 ms
64 bytes from 10.10.10.2: icmp_seq=14 ttl=64 time=6.75 ms
64 bytes from 10.10.10.2: icmp_seq=15 ttl=64 time=9.74 ms
64 bytes from 10.10.10.2: icmp_seq=16 ttl=64 time=7.05 ms
64 bytes from 10.10.10.2: icmp_seq=17 ttl=64 time=7.82 ms
64 bytes from 10.10.10.2: icmp_seq=18 ttl=64 time=7.63 ms
64 bytes from 10.10.10.2: icmp_seq=19 ttl=64 time=9.24 ms
64 bytes from 10.10.10.2: icmp_seq=20 ttl=64 time=9.25 ms

--- 10.10.10.2 ping statistics ---
20 packets transmitted, 20 received, 0% packet loss, time 19026ms
rtt min/avg/max/mdev = 5.158/26.753/225.281/57.193 ms

**Comment by MurphyMc** [comment]
I don't think you're running at DEBUG log level as I suggested.

Also, where are the hosts attached and what are their IPs?

**Comment by Maryam-Maya-Karimi** [comment]
h8 is 10.10.10.1 and is connected to s1
h9 is 10.10.10.2 and is connected to s3
here is the last  part of running with debug (all the part left in the terminal):

DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 404
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 438
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 409
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 363
DEBUG:openflow.discovery:Dropping LLDP packet 372
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 444
DEBUG:openflow.discovery:Dropping LLDP packet 445
DEBUG:openflow.discovery:Dropping LLDP packet 365
INFO:openflow.discovery:link detected: 00-00-00-00-00-05.2 -> 00-00-00-00-00-06.2
DEBUG:openflow.spanning_tree:Spanning tree updated
DEBUG:forwarding.l2_multi:Unlearned 82:f4:d4:02:a5:2e
DEBUG:forwarding.l2_multi:Unlearned 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Installing path for ea:69:a4:02:b6:ef -> 6e:cb:a1:3d:d9:9f 0800 (2 hops)
DEBUG:forwarding.l2_multi:Sending delayed packet out 00-00-00-00-00-01
DEBUG:forwarding.l2_multi:Installing path for 6e:cb:a1:3d:d9:9f -> ea:69:a4:02:b6:ef 0800 (2 hops)
DEBUG:forwarding.l2_multi:Sending delayed packet out 00-00-00-00-00-03
DEBUG:forwarding.l2_multi:Learned 02:cf:98:1f:ef:3d at 00-00-00-00-00-06.2
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 323
DEBUG:openflow.discovery:Dropping LLDP packet 416
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 451
DEBUG:openflow.discovery:Dropping LLDP packet 346
DEBUG:openflow.discovery:Dropping LLDP packet 378
DEBUG:openflow.discovery:Dropping LLDP packet 347
DEBUG:openflow.discovery:Dropping LLDP packet 419
DEBUG:openflow.discovery:Dropping LLDP packet 452
DEBUG:openflow.discovery:Dropping LLDP packet 420
DEBUG:openflow.discovery:Dropping LLDP packet 370
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 380
DEBUG:openflow.discovery:Dropping LLDP packet 454
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 455
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 373
DEBUG:openflow.discovery:Dropping LLDP packet 324
DEBUG:openflow.discovery:Dropping LLDP packet 423
DEBUG:openflow.discovery:Dropping LLDP packet 457
DEBUG:openflow.discovery:Dropping LLDP packet 349
DEBUG:openflow.discovery:Dropping LLDP packet 382
DEBUG:openflow.discovery:Dropping LLDP packet 350
DEBUG:openflow.discovery:Dropping LLDP packet 424
DEBUG:openflow.discovery:Dropping LLDP packet 458
DEBUG:openflow.discovery:Dropping LLDP packet 425
DEBUG:openflow.discovery:Dropping LLDP packet 374
DEBUG:openflow.discovery:Dropping LLDP packet 383
DEBUG:openflow.discovery:Dropping LLDP packet 459
DEBUG:openflow.discovery:Dropping LLDP packet 460
DEBUG:openflow.discovery:Dropping LLDP packet 375
DEBUG:openflow.discovery:Dropping LLDP packet 325
DEBUG:openflow.discovery:Dropping LLDP packet 426
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 462
DEBUG:openflow.discovery:Dropping LLDP packet 351
DEBUG:openflow.discovery:Dropping LLDP packet 385
DEBUG:openflow.discovery:Dropping LLDP packet 352
DEBUG:openflow.discovery:Dropping LLDP packet 428
DEBUG:openflow.discovery:Dropping LLDP packet 463
DEBUG:openflow.discovery:Dropping LLDP packet 429
DEBUG:openflow.discovery:Dropping LLDP packet 377
DEBUG:openflow.discovery:Dropping LLDP packet 386
DEBUG:forwarding.l2_multi:Learned 82:f4:d4:02:a5:2e at 00-00-00-00-00-05.2
DEBUG:forwarding.l2_multi:Flood multicast from 82:f4:d4:02:a5:2e
DEBUG:openflow.discovery:Dropping LLDP packet 464
DEBUG:forwarding.l2_multi:Flood multicast from 92:c0:c4:8f:ff:23
DEBUG:forwarding.l2_multi:Flood multicast from 92:c0:c4:8f:ff:23
DEBUG:forwarding.l2_multi:Flood multicast from 92:c0:c4:8f:ff:23
DEBUG:forwarding.l2_multi:Flood multicast from 92:c0:c4:8f:ff:23
DEBUG:forwarding.l2_multi:Flood multicast from 92:c0:c4:8f:ff:23
DEBUG:forwarding.l2_multi:Flood multicast from 92:c0:c4:8f:ff:23
DEBUG:forwarding.l2_multi:Flood multicast from 92:c0:c4:8f:ff:23
DEBUG:forwarding.l2_multi:Flood multicast from 82:6b:63:b1:ad:5d
DEBUG:forwarding.l2_multi:Flood multicast from fe:bc:e6:c0:ca:e3
DEBUG:forwarding.l2_multi:Flood multicast from 66:4c:0a:f5:2a:7b
DEBUG:forwarding.l2_multi:Flood multicast from 66:4c:0a:f5:2a:7b
DEBUG:forwarding.l2_multi:Flood multicast from 0a:4c:f4:29:cf:fe
DEBUG:forwarding.l2_multi:Flood multicast from 0a:4c:f4:29:cf:fe
DEBUG:forwarding.l2_multi:Flood multicast from 0a:4c:f4:29:cf:fe
DEBUG:forwarding.l2_multi:Flood multicast from 9a:de:39:0f:5e:bc
DEBUG:forwarding.l2_multi:Flood multicast from 9a:de:39:0f:5e:bc
DEBUG:forwarding.l2_multi:Flood multicast from 3e:17:c8:74:9e:3b
DEBUG:forwarding.l2_multi:Flood multicast from 9a:de:39:0f:5e:bc
DEBUG:forwarding.l2_multi:Flood multicast from 3e:17:c8:74:9e:3b
DEBUG:forwarding.l2_multi:Flood multicast from 3e:17:c8:74:9e:3b
DEBUG:forwarding.l2_multi:Flood multicast from 3e:17:c8:74:9e:3b
DEBUG:forwarding.l2_multi:Flood multicast from 3e:17:c8:74:9e:3b
DEBUG:forwarding.l2_multi:Flood multicast from 2a:c2:03:29:4b:d4
DEBUG:forwarding.l2_multi:Flood multicast from 3e:17:c8:74:9e:3b
DEBUG:forwarding.l2_multi:Flood multicast from 76:ee:40:2f:e1:b7
DEBUG:forwarding.l2_multi:Flood multicast from ce:3f:38:a4:54:f0
DEBUG:forwarding.l2_multi:Flood multicast from 76:ee:40:2f:e1:b7
DEBUG:forwarding.l2_multi:Flood multicast from ce:3f:38:a4:54:f0
DEBUG:forwarding.l2_multi:Flood multicast from 76:ee:40:2f:e1:b7
DEBUG:forwarding.l2_multi:Flood multicast from ce:3f:38:a4:54:f0
DEBUG:forwarding.l2_multi:Flood multicast from 76:ee:40:2f:e1:b7
DEBUG:forwarding.l2_multi:Flood multicast from ce:3f:38:a4:54:f0
DEBUG:openflow.discovery:Dropping LLDP packet 383
DEBUG:forwarding.l2_multi:Flood multicast from 76:ee:40:2f:e1:b7
DEBUG:forwarding.l2_multi:Flood multicast from ce:3f:38:a4:54:f0
DEBUG:forwarding.l2_multi:Flood multicast from 76:ee:40:2f:e1:b7
DEBUG:forwarding.l2_multi:Flood multicast from ce:3f:38:a4:54:f0
DEBUG:forwarding.l2_multi:Flood multicast from 52:84:7b:57:f6:12
DEBUG:openflow.discovery:Dropping LLDP packet 334
DEBUG:openflow.discovery:Dropping LLDP packet 437
DEBUG:openflow.discovery:Dropping LLDP packet 469
DEBUG:openflow.discovery:Dropping LLDP packet 356
DEBUG:openflow.discovery:Dropping LLDP packet 391
DEBUG:openflow.discovery:Dropping LLDP packet 357
DEBUG:openflow.discovery:Dropping LLDP packet 438
DEBUG:openflow.discovery:Dropping LLDP packet 470
DEBUG:openflow.discovery:Dropping LLDP packet 439
DEBUG:openflow.discovery:Dropping LLDP packet 386
DEBUG:openflow.discovery:Dropping LLDP packet 392
INFO:openflow.discovery:link timeout: 00-00-00-00-00-03.3 -> 00-00-00-00-00-01.3
DEBUG:openflow.spanning_tree:Spanning tree updated
DEBUG:openflow.discovery:Dropping LLDP packet 471
DEBUG:openflow.discovery:Dropping LLDP packet 387
DEBUG:openflow.discovery:Dropping LLDP packet 335
DEBUG:openflow.discovery:Dropping LLDP packet 440
DEBUG:openflow.discovery:Dropping LLDP packet 472
DEBUG:openflow.discovery:Dropping LLDP packet 357
DEBUG:openflow.discovery:Dropping LLDP packet 393
DEBUG:openflow.discovery:Dropping LLDP packet 358
DEBUG:openflow.discovery:Dropping LLDP packet 441
DEBUG:openflow.discovery:Dropping LLDP packet 473
DEBUG:openflow.discovery:Dropping LLDP packet 442
DEBUG:openflow.discovery:Dropping LLDP packet 388
DEBUG:openflow.discovery:Dropping LLDP packet 394
DEBUG:openflow.discovery:Dropping LLDP packet 474
DEBUG:openflow.discovery:Dropping LLDP packet 389
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 336
DEBUG:openflow.discovery:Dropping LLDP packet 444
DEBUG:forwarding.l2_multi:Installing path for ea:69:a4:02:b6:ef -> 6e:cb:a1:3d:d9:9f 0800 (2 hops)
DEBUG:forwarding.l2_multi:Sending delayed packet out 00-00-00-00-00-01
DEBUG:forwarding.l2_multi:Installing path for 6e:cb:a1:3d:d9:9f -> ea:69:a4:02:b6:ef 0800 (2 hops)
DEBUG:forwarding.l2_multi:Sending delayed packet out 00-00-00-00-00-03
DEBUG:openflow.discovery:Dropping LLDP packet 476
DEBUG:openflow.discovery:Dropping LLDP packet 358
DEBUG:openflow.discovery:Dropping LLDP packet 395
DEBUG:openflow.discovery:Dropping LLDP packet 359
DEBUG:openflow.discovery:Dropping LLDP packet 446
DEBUG:openflow.discovery:Dropping LLDP packet 477
DEBUG:openflow.discovery:Dropping LLDP packet 447
DEBUG:openflow.discovery:Dropping LLDP packet 391
DEBUG:openflow.discovery:Dropping LLDP packet 396
DEBUG:openflow.discovery:Dropping LLDP packet 478
DEBUG:openflow.discovery:Dropping LLDP packet 392
DEBUG:openflow.discovery:Dropping LLDP packet 337
DEBUG:forwarding.l2_multi:Installing path for 6e:cb:a1:3d:d9:9f -> ea:69:a4:02:b6:ef 0806 (2 hops)
DEBUG:forwarding.l2_multi:Sending delayed packet out 00-00-00-00-00-03
DEBUG:openflow.discovery:Dropping LLDP packet 449
DEBUG:openflow.discovery:Dropping LLDP packet 479
DEBUG:openflow.discovery:Dropping LLDP packet 359
DEBUG:openflow.discovery:Dropping LLDP packet 397
DEBUG:openflow.discovery:Dropping LLDP packet 360
DEBUG:openflow.discovery:Dropping LLDP packet 450
DEBUG:openflow.discovery:Dropping LLDP packet 480
DEBUG:openflow.discovery:Dropping LLDP packet 451
DEBUG:openflow.discovery:Dropping LLDP packet 393
DEBUG:openflow.discovery:Dropping LLDP packet 398
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:openflow.discovery:Dropping LLDP packet 481
DEBUG:openflow.discovery:Dropping LLDP packet 395
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:openflow.discovery:Dropping LLDP packet 340
DEBUG:openflow.discovery:Dropping LLDP packet 454
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:openflow.discovery:Dropping LLDP packet 482
DEBUG:openflow.discovery:Dropping LLDP packet 360
DEBUG:openflow.discovery:Dropping LLDP packet 399
DEBUG:openflow.discovery:Dropping LLDP packet 361
DEBUG:openflow.discovery:Dropping LLDP packet 456
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:openflow.discovery:Dropping LLDP packet 483
DEBUG:openflow.discovery:Dropping LLDP packet 458
DEBUG:openflow.discovery:Dropping LLDP packet 399
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:openflow.discovery:Dropping LLDP packet 400
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:openflow.discovery:Dropping LLDP packet 484
DEBUG:openflow.discovery:Dropping LLDP packet 402
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:openflow.discovery:Dropping LLDP packet 346
DEBUG:openflow.discovery:Dropping LLDP packet 462
DEBUG:openflow.discovery:Dropping LLDP packet 485
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:openflow.discovery:Dropping LLDP packet 361
DEBUG:openflow.discovery:Dropping LLDP packet 401
DEBUG:openflow.discovery:Dropping LLDP packet 362
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:openflow.discovery:Dropping LLDP packet 465
DEBUG:openflow.discovery:Dropping LLDP packet 486
DEBUG:openflow.discovery:Dropping LLDP packet 466
DEBUG:openflow.discovery:Dropping LLDP packet 406
DEBUG:openflow.discovery:Dropping LLDP packet 402
DEBUG:openflow.discovery:Dropping LLDP packet 487
DEBUG:openflow.discovery:Dropping LLDP packet 407
DEBUG:openflow.discovery:Dropping LLDP packet 349
DEBUG:openflow.discovery:Dropping LLDP packet 467
DEBUG:openflow.discovery:Dropping LLDP packet 488
DEBUG:openflow.discovery:Dropping LLDP packet 362
DEBUG:openflow.discovery:Dropping LLDP packet 403
DEBUG:openflow.discovery:Dropping LLDP packet 363
DEBUG:openflow.discovery:Dropping LLDP packet 468
DEBUG:openflow.discovery:Dropping LLDP packet 489
DEBUG:openflow.discovery:Dropping LLDP packet 469
DEBUG:openflow.discovery:Dropping LLDP packet 408
DEBUG:openflow.discovery:Dropping LLDP packet 404
DEBUG:openflow.discovery:Dropping LLDP packet 490
DEBUG:openflow.discovery:Dropping LLDP packet 409
DEBUG:openflow.discovery:Dropping LLDP packet 350
DEBUG:openflow.discovery:Dropping LLDP packet 470
DEBUG:openflow.discovery:Dropping LLDP packet 491
DEBUG:openflow.discovery:Dropping LLDP packet 363
DEBUG:openflow.discovery:Dropping LLDP packet 405
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Installing path for ea:69:a4:02:b6:ef -> 6e:cb:a1:3d:d9:9f 0800 (2 hops)
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Sending delayed packet out 00-00-00-00-00-01
^ADEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Flood multicast from 6e:cb:a1:3d:d9:9f
DEBUG:forwarding.l2_multi:Installing path for ea:69:a4:02:b6:ef -> 6e:cb:a1:3d:d9:9f 0806 (2 hops)
DEBUG:forwarding.l2_multi:Sending delayed packet out 00-00-00-00-00-01
DEBUG:forwarding.l2_multi:Installing path for 6e:cb:a1:3d:d9:9f -> ea:69:a4:02:b6:ef 0800 (2 hops)
DEBUG:openflow.discovery:Dropping LLDP packet 366
DEBUG:forwarding.l2_multi:Sending delayed packet out 00-00-00-00-00-03
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:openflow.discovery:Dropping LLDP packet 473
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:openflow.discovery:Dropping LLDP packet 498
DEBUG:openflow.discovery:Dropping LLDP packet 474
DEBUG:openflow.discovery:Dropping LLDP packet 411
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:openflow.discovery:Dropping LLDP packet 416
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from 02:cf:98:1f:ef:3d
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:openflow.discovery:Dropping LLDP packet 509
INFO:openflow.discovery:link detected: 00-00-00-00-00-03.3 -> 00-00-00-00-00-01.3
DEBUG:openflow.spanning_tree:Spanning tree updated
DEBUG:forwarding.l2_multi:Unlearned c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Unlearned 0a:4c:f4:29:cf:fe
DEBUG:forwarding.l2_multi:Learned c6:8c:a7:57:fa:cb at 00-00-00-00-00-01.3
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:openflow.discovery:Dropping LLDP packet 512
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Installing path for ea:69:a4:02:b6:ef -> 6e:cb:a1:3d:d9:9f 0800 (2 hops)
DEBUG:forwarding.l2_multi:Sending delayed packet out 00-00-00-00-00-01
DEBUG:forwarding.l2_multi:Installing path for 6e:cb:a1:3d:d9:9f -> ea:69:a4:02:b6:ef 0800 (2 hops)
DEBUG:forwarding.l2_multi:Sending delayed packet out 00-00-00-00-00-03
DEBUG:openflow.discovery:Dropping LLDP packet 413
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:openflow.discovery:Dropping LLDP packet 352
DEBUG:openflow.discovery:Dropping LLDP packet 477
DEBUG:openflow.discovery:Dropping LLDP packet 517
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:openflow.discovery:Dropping LLDP packet 384
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:openflow.discovery:Dropping LLDP packet 428
DEBUG:openflow.discovery:Dropping LLDP packet 386
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:forwarding.l2_multi:Flood multicast from c6:8c:a7:57:fa:cb
DEBUG:openflow.discovery:Dropping LLDP packet 478
DEBUG:openflow.discovery:Dropping LLDP packet 522
DEBUG:openflow.discovery:Dropping LLDP packet 479
DEBUG:openflow.discovery:Dropping LLDP packet 414
DEBUG:openflow.discovery:Dropping LLDP packet 431
DEBUG:openflow.discovery:Dropping LLDP packet 523
DEBUG:openflow.discovery:Dropping LLDP packet 524

**Comment by MurphyMc** [comment]
This may be a regression caused by 1216d5f812f4c59b8f86b9cdf90520c9d3cfd7f0.

In l2_multi around line 457, try adding a line which skips the removed link in discovery's adjacency list, like this:

```
      # But maybe there's another way to connect these...
      for ll in core.openflow_discovery.adjacency:
        if ll == event.link: continue # <-- Add this line
```

**Comment by Maryam-Maya-Karimi** [comment]
thanks a lot for quick reply. it is solved, thanks a lot :)

**Comment by MurphyMc** [comment]
This is now fixed in eel.  Thanks for the bug report (though in the future, please start a new issue for new issues).

**Comment by Maryam-Maya-Karimi** [comment]
alright. thanks again.
